shared/src/lib.rs

```diff
@@ -62,51 +45,11 @@ pub const IDENTITY_HASH: u64 = 0x0;
 /// The maximum supported CID size.
 pub const MAX_CID_LEN: usize = 100;
 
-/// Identifier for Actors, includes builtin and initialized actors
-pub type ActorID = u64;
-
 /// Default bit width for the hamt in the filecoin protocol.
 pub const HAMT_BIT_WIDTH: u32 = 5;
-/// Total gas limit allowed per block. This is shared across networks.
-pub const BLOCK_GAS_LIMIT: u64 = 10_000_000_000;
```

Comment:
Is this only enforced in the miner at message selection time and in nodes validating messages in blocks packed by miners? Otherwise this isn't interesting anywhere else?

Reply:
Yes. We only enforce per-message gas limits in the FVM. Otherwise cron would be an issue.

Reply:
Really, the FVM doesn't even see "blocks", it just sees a bunch of messages to be applied at a specific height on top of a previous state-tree.